sdk/identity/Azure.Identity/src/Credentials/ClientAssertionCredential.cs

```diff
@@ -100,6 +102,8 @@ public override AccessToken GetToken(TokenRequestContext requestContext, Cancell
         /// <param name="requestContext">The details of the authentication request.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>An <see cref="AccessToken"/> which can be used to authenticate service client calls.</returns>
+        /// <exception cref="AuthenticationFailedException">Thrown when the authentication failed.</exception>
+        /// <exception cref="CredentialUnavailableException">Thrown when the credential is unavailable when used as part of a chained credential such as <see cref="DefaultAzureCredential"/> or <see cref="ChainedTokenCredential"/>.</exception>
```

Comment:
I think this wording could be confusing, since ClientAssertionCredential is indirectly included in DAC via EnvironmentCredential.